app/javascript/components/atoms/Spinner.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+const Spinner = ({  }) => {
```

Comment:
I would replace `({  })`  for just `()`  You probably saw I was doing that on one of my components, but since we probably are going to implement eslint soon, it's going to complain about it.